product-scenarios/code-coverage/code-coverage.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+# Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#=== FUNCTION ==================================================================
+# NAME: generate_code_coverage
+# DESCRIPTION: Genaret code coverage reports.
+# PARAMETER 1: Path to input directory
+# PARAMETER 2: Path to output directory
+#===============================================================================
+
+#set -o xtrace
+
+FILE=""
+VAR_TEST_PLAN_ID=""
+VAR_IS_TESTGRID=""
+VAR_TINKERER_ENDPOINT=""
+VAR_TINKERER_USERNAME=""
+VAR_TINKERER_PASSWORD=""
+VAR_REM_DIR=""
+
+function generate_code_coverage(){
+    INPUT_DIR=$1
+    OUTPUT_DIR=$2
+    HOME=`pwd`
+
+    FILE="${INPUT_DIR}/deployment.properties"
+    PROP_TEST_PLAN_ID=TEST_PLAN_ID
+    PROP_TESTGRID=IS_TESTGRID
+    PROP_TINKERE_ENDPOINT=TESTGRID_TINKERER_ENDPOINT
+    PROP_SERVER_DIR=RemoteProductDir
+    PROP_TINKERER_USERNAME=TESTGRID_TINKERER_USERNAME
+    PROP_TINKERER_PASSWORD=TESTGRID_TINKERER_PASSWORD
+    user=ubuntu
+
+    #=============== Read Deployment.property file ===============================================
+
+    VAR_TEST_PLAN_ID=`cat ${FILE} | grep -w "$PROP_TEST_PLAN_ID" ${FILE} | cut -d'=' -f2`
```

Comment:
Shall we use https://github.com/wso2/product-apim/blob/product-scenarios-dev/product-scenarios/test.sh#L85-L88 function for this?